src/context/auth-context/auth-context.js

```diff
@@ -20,12 +20,14 @@ function AuthProvider({ children }) {
           const response = await axios.post("/api/auth/verify", {
```

Comment:
On line 12 you are using 2 use states for the user and encodedToken make it in one using the object.

Reply:
will check